### PR TITLE
First release

### PR DESCRIPTION
**Pull request by luisas**
PR for first release 

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/multiplesequencealign/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, also make a PR on the nf-core/multiplesequencealign _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by mirpedrol — COMMENTED** [review]
Posting comments from a first review, will continue tomorrow 😄

File: tests/nextflow.config [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    withName: '.*' {
+        cpus   = 2
+        memory = 3.GB
+        time   = 2.h
+    }
+}
+
+// Impose same minimum Nextflow version as the pipeline for testing
+manifest {
+    nextflowVersion = '!>=23.04.0'
```
Comment (mirpedrol) [inline tests/nextflow.config:22]: You used `>=24.04.1` on the pipeline `nextflow.config`

File: workflows/multiplesequencealign.nf [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+    // Dependency files are taken from a directory.
+    // If the directory is compressed, it is uncompressed first.
+    ch_dependencies
+        .branch {
+            compressed:   it[1].endsWith('.tar.gz')
+            uncompressed: true
+        }
+        .set { ch_dependencies }
+
+    UNTAR (ch_dependencies.compressed)
```
Comment (mirpedrol) [inline workflows/multiplesequencealign.nf:114]: Versions are not collected for this module

**Review by mirpedrol — APPROVED** [review]
Some last comments, otherwise looks good! :)

File: assets/multiqc_config.yml [hunk 3]
```diff
@@ -1,13 +1,166 @@
 report_comment: >
-  This report has been generated by the <a href="https://github.com/nf-core/msa/1.0dev" target="_blank">nf-core/msa</a>
+  This report has been generated by the <a href="https://github.com/nf-core/multiplesequencealign/releases/tag/1.0.0" target="_blank">nf-core/multiplesequencealign</a>
   analysis pipeline. For information about how to interpret these results, please see the
-  <a href="https://nf-co.re/msa/1.0dev/output" target="_blank">documentation</a>.
+  <a href="https://nf-co.re/multiplesequencealign/0.1.0dev/docs/output" target="_blank">documentation</a>.
```
[the remark this answers is not in the document]
Comment (luisas) [reply assets/multiqc_config.yml:4]: done

File: workflows/multiplesequencealign.nf (hunk 2 above)
Comment (luisas) [reply workflows/multiplesequencealign.nf:114]: done

File: tests/nextflow.config (hunk 1 above)
Comment (luisas) [reply tests/nextflow.config:22]: ups, updated here too

File: nextflow.config [hunk 4]
```diff
@@ -180,15 +217,10 @@ singularity.registry = 'quay.io'
 
 // Nextflow plugins
 plugins {
-    id 'nf-validation' // Validation of pipeline parameters and creation of an input channel from a sample sheet
+    id 'nf-validation@1.1.3' // Validation of pipeline parameters and creation of an input channel from a sample sheet
```
[the remark this answers is not in the document]
Comment (luisas) [reply nextflow.config:220]: updated

File: nextflow.config [hunk 5]
```diff
@@ -1,40 +1,66 @@
...
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    nf-core/msa Nextflow config file
+    nf-core/multiplesequencealign Nextflow config file
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     Default config options for all compute environments
 ----------------------------------------------------------------------------------------
 */
 
+plugins {
+    id 'nf-validation@0.3.1'
```
[the remark this answers is not in the document]
Comment (luisas) [reply nextflow.config:10]: ermoved

File: modules/nf-core/custom/dumpsoftwareversions/templates/dumpsoftwareversions.py
[the remark this answers is not in the document]
Comment (luisas) [reply modules/nf-core/custom/dumpsoftwareversions/templates/dumpsoftwareversions.py:1]: done

File: assets/schema_tools.json [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+    "$id": "https://raw.githubusercontent.com/nf-core/msa/master/assets/schema_tools.json",
+    "title": "nf-core/msa pipeline - params.tools schema",
+    "description": "Schema for the file provided with params.tools",
+    "type": "array",
+    "items": {
+        "type": "object",
+        "properties": {
+            "tree": {
+                "type": "string",
+                "errorMessage": "tree name cannot contain spaces",
```
[the remark this answers is not in the document]
Comment (luisas) [reply assets/schema_tools.json:12]: uh yes! thanks :)

File: assets/schema_tools.json [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+                "meta": ["tree"]
+            },
+            "args_tree": {
+                "type": "string",
+                "meta": ["args_tree"]
+            },
+            "aligner": {
+                "type": "string",
+                "meta": ["aligner"],
+                "errorMessage": "align name must be provided and cannot contain spaces"
```
[the remark this answers is not in the document]
Comment (luisas) [reply assets/schema_tools.json:22]: added!

File: docs/usage.md [hunk 8]
```diff
@@ -99,31 +201,34 @@ You can also generate such `YAML`/`JSON` files via [nf-core/launch](https://nf-c
...
+> These options are part of Nextflow and use a _single_ hyphen (pipeline parameters use a double-hyphen).
 
 ### `-profile`
 
 Use this parameter to choose a configuration profile. Profiles can give configuration presets for different compute environments.
 
 Several generic profiles are bundled with the pipeline which instruct the pipeline to use software packaged using different methods (Docker, Singularity, Podman, Shifter, Charliecloud, Apptainer, Conda) - see below.
 
+> [!INFO]
 > We highly recommend the use of Docker or Singularity containers for full pipeline reproducibility, however when this is not possible, Conda is also supported.
```
[the remark this answers is not in the document]
Comment (luisas) [reply docs/usage.md:232]: Updated

File: modules/nf-core/csvtk/join/main.nf [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    prefix = task.ext.prefix ?: "${meta.id}"
+    out_extension = args.contains('--out-delimiter "\t"') || args.contains('-D "\t"') || args.contains("-D \$'\t'") ? "tsv" : "csv"
+    """
+    
+    # if the input is horter than 2, add an emtpy file as the second input
```
[the remark this answers is not in the document]
Comment (luisas) [reply modules/nf-core/csvtk/join/main.nf:26]: We don't, you are right! I updated the comment correspondingly :)

File: assets/adaptivecard.json [hunk 10]
```diff
@@ -17,7 +17,7 @@
                         "size": "Large",
                         "weight": "Bolder",
                         "color": "<% if (success) { %>Good<% } else { %>Attention<%} %>",
-                        "text": "nf-core/msa v${version} - ${runName}",
+                        "text": "nf-core/multiplesequencealign v${version} - ${runName}",
```
Comment (maxulysse) [inline assets/adaptivecard.json:20]:
```suggestion
                        "text": "nf-core/multiplesequencealign ${version} - ${runName}",
```

Comment (maxulysse) [reply assets/adaptivecard.json:20]: `assets/slackreport.json` and `assets/email_template.html` don't have the `v`

File: README.md [hunk 11]
```diff
@@ -1,99 +1,124 @@
...
+[![Get help on Slack](http://img.shields.io/badge/slack-nf--core%20%23multiplesequencealign-4A154B?labelColor=000000&logo=slack)](https://nfcore.slack.com/channels/multiplesequencealign)[![Follow on Twitter](http://img.shields.io/badge/twitter-%40nf__core-1DA1F2?labelColor=000000&logo=twitter)](https://twitter.com/nf_core)[![Follow on Mastodon](https://img.shields.io/badge/mastodon-nf__core-6364ff?labelColor=FFFFFF&logo=mastodon)](https://mstdn.science/@nf_core)[![Watch on YouTube](http://img.shields.io/badge/youtube-nf--core-FF0000?labelColor=000000&logo=youtube)](https://www.youtube.com/c/nf-core)
 
 ## Introduction
 
-**nf-core/msa** is a bioinformatics pipeline that ...
+**nf-core/multiplesequencealign** is a pipeline to deploy and systematically evaluate Multiple Sequence Alignment (MSA) methods.
```
Comment (maxulysse) [inline README.md:22]:
```suggestion
**nf-core/multiplesequencealign** is a pipeline to deploy and systematically evaluate Multiple Sequence Alignment (MSA) methods.

The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool to run tasks across multiple compute infrastructures in a very portable manner. It uses Docker/Singularity containers making installation trivial and results highly reproducible. The [Nextflow DSL2](https://www.nextflow.io/docs/latest/dsl2.html) implementation of this pipeline uses one container per process which makes it much easier to maintain and update software dependencies. Where possible, these processes have been submitted to and installed from [nf-core/modules](https://github.com/nf-core/modules) in order to make them available to all nf-core pipelines, and to everyone within the Nextflow community!

On release, automated continuous integration tests run the pipeline on a full-sized dataset on the AWS cloud infrastructure. This ensures that the pipeline runs on AWS, has sensible resource allocation defaults set to run on real-world datasets, and permits the persistent storage of results to benchmark between pipeline releases and other analysis sources.The results obtained from the full-sized test can be viewed on the [nf-core website](https://nf-co.re/multiplesequencealign/results).```

Still missing from TEMPLATE

File: docs/images/nf-core-msa_metro_map.png
Comment (maxulysse) [inline docs/images/nf-core-msa_metro_map.png:1]: I'd recommend not to have the version in the map as it's the one thing people tend to forget

File: tests/nextflow.config (hunk 1 above)
Comment (maxulysse) [reply tests/nextflow.config:22]: where was this updated?

File: assets/adaptivecard.json (hunk 10 above)
Comment (luisas) [reply assets/adaptivecard.json:20]: Ok - i will put your suggestion in another PR together with Julia's ones :)

File: README.md (hunk 11 above)
Comment (luisas) [reply README.md:22]: updated on the PR

Comment (luisas) [reply README.md:22]: [PR](https://github.com/nf-core/multiplesequencealign/pull/161
)

File: tests/nextflow.config (hunk 1 above)
Comment (luisas) [reply tests/nextflow.config:22]: here --> https://github.com/nf-core/multiplesequencealign/pull/161

File: docs/images/nf-core-msa_metro_map.png
Comment (luisas) [reply docs/images/nf-core-msa_metro_map.png:1]: uh good one - removed it!

**Review by maxulysse — APPROVED** [review]

**Pull request merged**